common/compiler/helpers.go

```diff
@@ -21,9 +21,12 @@ import (
 	"bytes"
 	"io/ioutil"
 	"regexp"
+
+	semver "github.com/Masterminds/semver/v3"
 )
 
 var versionRegexp = regexp.MustCompile(`([0-9]+)\.([0-9]+)\.([0-9]+)`)
+var pragmaVersionRegexp = regexp.MustCompile(`(?m)^pragma solidity (.+)\;$`)
```

Comment:
This regex didn't consider whitespaces(ws) and failed if the input string contains multiple ws between words.
Consider change with this one:
```
regexp.MustCompile(`(?m)^(\s*)pragma(\s*) solidity(\s*) (.+)\;(\s*)$`)
```
Now this regex have interesting matched target is in the index 4. (i.e., line78 should be fixed too) (0 = input string, 1 = \s+, 2 = \s+, 3 = \s+ 4 = matched version). 

FYI, the regex is not the correct implementation if this regex is solely used since any string would be matched by `(.+)`. However, this incorrectly matched string would be filtered at `solcCanCompile()` function by semver package if the result of the regex is passed to this function before being used.